.osdk-scorecard.yaml

```diff
@@ -1,4 +1,4 @@
 cr-manifest: "deploy/crds/tekton_v1alpha1_install_cr.yaml"
 init-timeout: 60
-csv-path: "deploy/olm-catalog/tektoncd-pipeline-operator/0.0.1/tektoncd-pipeline-operator.v0.0.1.clusterserviceversion.yaml"
-
+csv-path: "deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml"
```

Comment:
shouldn't this be 'openshift-pipeline-operator' as well? 
@nikhil-thomas Could you please look for occurrence of tektoncd-pipeline and see if that can be replaced with openshift-pipeline ?

Reply:
i can do that, but that means changing it in crds, csv names, package names etc which are generated form repository name (and break conventions)

That may ruin developer experience when somebody is trying to contribute/collaborate
Shall we enforce the elimination of 'TektonCD' when we move the org. Then we can name downstream repo 'openshift-pipeline-operator'. Then the renaming will make more sense.

Reply:
done